Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [GnuCash/Program](https://hosted.weblate.org/projects/gnucash/gnucash/).


It also includes following components:

* [GnuCash/Glossary](https://hosted.weblate.org/projects/gnucash/glossary/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/gnucash/gnucash/horizontal-auto.svg)

fellen commented on po/de.po:
```diff
@@ -18610,7 +18610,7 @@ msgstr "Ausführliche _Fehlermeldungen"
 #: gnucash/import-export/aqb/gschemas/org.gnucash.GnuCash.dialogs.import.hbci.gschema.xml.in:31
 msgid "Enables verbose debug messages for HBCI/AqBanking Online Banking."
-msgstr "Ausführliche Fehlermeldungen für HBCI Online Banking aktivieren."
+msgstr "Ausführliche Fehlermeldungen für das FinTS Online Banking aktivieren."
```
https://wiki.gnucash.org/wiki/AqBanking#Debugging: Since Gnucash 2.3.x this is related to AQBANKING_LOGLEVEL.
also für ganz AqBanking

fellen replied:
"FinTS/" weg.

fellen commented on po/de.po:
```diff
@@ -18875,11 +18875,11 @@ msgstr "Online-Auftrag SEPA-Überweisung"
 #: gnucash/import-export/aqb/gnc-ab-transfer.c:218
 msgid "Online Banking European (SEPA) Internal Transfer"
-msgstr "Online-Auftrag SEPA-Übertrag"
+msgstr "Online-Auftrag interne SEPA-Umbuchung"
```
Bank-interne Umbuchung.
Die alte, nationale Methode wird, zumindest in FinTS, vermutlich aber auch EBICs, nicht mehr unterstützt. Von daher könnte man die Einträge alle vereinfachen (erst im Quelltext, dann in den Po's)

fellen commented on po/de.po:
```diff
@@ -18584,12 +18584,12 @@ msgid ""
 "AqBanking import process. Otherwise it will stay open."
 msgstr ""
-"Wenn aktiviert, wird das Fenster automatisch geschlossen, wenn der HBCI/"
+"Wenn aktiviert, wird das Fenster automatisch geschlossen, sobald der FinTS/"
```
Ich glaube, das Fenster wird auch von den anderen Aqbanking-Modulen benutzt.
[€dit:]
Also "FinTS/" weg.

fellen commented on po/de.po:
```diff
@@ -5885,8 +5885,8 @@ msgid ""
 "Exchange by CSV/FinTS(former HBCI) or import SWIFT-MT9xx/QIF/OFX data "
 "including Transaction Matching"
 msgstr ""
-"Datenaustausch mittels CSV/FinTS(former HBCI) oder importiere SWIFT-MT9xx/"
-"QIF/OFX Daten, einschließlich Buchungsabgleich"
+"Datenaustausch mittels CSV/FinTS oder importiere SWIFT-MT9xx/QIF/OFX Daten, "
+"einschließlich Buchungsabgleich"
```
Christian, schreib doch bitte in solchen Fällen Kommentare gegen den Quelltext rein. Dann können wir die Liste Peu a Peu abarbeiten.

fellen commented:
Thanks!

CWehli committed:
Translation update  by Christian Wehling <christian.wehling@web.de> using Weblate

po/de.po: 100.0% (5540 of 5540 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (German)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/de/

Co-authored-by: Christian Wehling <christian.wehling@web.de>
